packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6879,6 +6879,10 @@ export default class Meeting extends StatelessWebexPlugin {
     this.audio = null;
     this.video = null;
     this.screenShareFloorState = ScreenShareFloorStatus.RELEASED;
+    this.sendSlotManager.deleteSlot(MediaType.AudioMain);
```

Comment:
We create the send slots in createMediaConnection() and the WCME send slots are closely coupled with the WCME MultistreamConnection, so I think we should be deleting them when we delete the WCME MultistreamConnection, because they won't work without it. So probably a better place to delete them is in `closePeerConnections()` - just after calling `this.mediaProperties.webrtcMediaConnection.close()`

Reply:
Done @marcin-bazyl. Also as mentioned in the 1:1 chat, have removed the deleteSlot method and introduced a one-reset method

Kindly check